src/makeRobotAccountant.js

```diff
@@ -6,7 +6,26 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let sum = 0;
+  let counter = 0;
+
+  const getSum = (firstNumber) => {
+    const getSecondNumber = (secondNumber) => {
+      counter++;
+
+      if (counter > 3 && counter % 2 === 0) {
+        return `Bzzz... Error!`;
+      }
+
+      sum = firstNumber + secondNumber;
+
+      return sum;
```

Comment:
you can return value immediately
```suggestion
      return firstNumber + secondNumber;
```